src/InstrumentationEngine.Attach/Program.cs

```diff
@@ -0,0 +1,170 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Text.Json;
+using Microsoft.Diagnostics.NETCore.Client;
+using static System.FormattableString;
+
+namespace Microsoft.InstrumentationEngine
+{
+    internal sealed class Program
+    {
+        private static readonly Guid InstrumentationEngineClsid = new Guid("324F817A-7420-4E6D-B3C1-143FBED6D855");
+
+        internal static void Main(string[] args)
```

Comment:
>void [](start = 24, length = 4)

Should change this to int and return a non-zero exit code if any failure occurs.